site/src/modules/apps/useAppLink.ts

```diff
@@ -58,15 +58,17 @@ export const useAppLink = (
 			const openAppExternallyFailedTimeout = 500;
 			const openAppExternallyFailed = setTimeout(() => {
 				// Check if this is a JetBrains IDE app
-				const isJetBrainsApp =
-					app.url &&
-					(app.url.startsWith("jetbrains-gateway:") ||
-						app.url.startsWith("jetbrains:"));
+				const isJetBrainsGateway = app.url?.startsWith("jetbrains-gateway:"); // starts with "jetbrains-gateway://connect#type=coder" (from https://registry.coder.com/modules/coder/jetbrains-gateway)
+				const isJetBrainsToolbox = app.url?.startsWith("jetbrains:"); // starts with "jetbrains://gateway/coder" (from https://registry.coder.com/modules/coder/jetbrains)
```

Comment:
nit: this read better to me

```suggestion
				// starts with "jetbrains-gateway://connect#type=coder" (from https://registry.coder.com/modules/coder/jetbrains-gateway)
				const isJetBrainsGateway = app.url?.startsWith("jetbrains-gateway:");
				// starts with "jetbrains://gateway/coder" (from https://registry.coder.com/modules/coder/jetbrains)
				const isJetBrainsToolbox = app.url?.startsWith("jetbrains:");
```